src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -118,6 +131,12 @@
                 removeClass(ajaxToolbarPanel, 'sf-ajax-request-loading');
                 removeClass(ajaxToolbarPanel, 'sf-toolbar-status-red');
             }
+
+            addEventListener(document.querySelector('.sf-toolbar-ajax-clear'), 'click', function() {
+                requestStack = [];
+                renderAjaxRequests();
```

Comment:
you need to reset the toolbar status color
Adding `successStreak = 4;` should do the trick